tests/storage/utils.py

```
@@ -281,40 +282,48 @@ def get_importer_pod(
+            if event_type == "DELETED":
+                LOGGER.warning(f"DataVolume {dv.name} was deleted while waiting for message: {expected_message}")
+                continue
+            if event_type not in ("ADDED", "MODIFIED"):
+                LOGGER.info(f"Ignoring {event_type} event for DataVolume {dv.name}")
+                continue
+            conditions = (event["object"].status or {}).get("conditions", [])
+            if any(expected_message in condition.get("message", "") for condition in conditions):
+                LOGGER.info(f"Found expected message in {dv.name}: {expected_message}")
+                return
+    except Exception as exc:
+        LOGGER.error(f"Error while watching DataVolume {dv.name}: {exc}")
```

Ruff (0.15.12)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)